lib/src/modules/calendar/Calendar.js

```diff
@@ -2,20 +2,36 @@ import React from 'react';
 import DayPicker from 'react-day-picker';
 import moment from 'moment';
 
-export function Calendar({ onDayClick, selectedDay, className }) {
+export function Calendar({
+  onDayClick,
+  selectedDay,
+  className,
+  enableWeekendDays = false,
+  leftAlignHeader = false
+}) {
   const today = moment().set({ hour: 0, minute: 0 });
 
   const modifiers = {
-    weekend: {
-      daysOfWeek: [0, 6]
-    },
     past: day => day < today
   };
 
+  if (enableWeekendDays === false) {
+    modifiers.weekend = {
+      daysOfWeek: [0, 6]
+    };
+  }
+
   const weekdayText = ['Sun', 'Mon', 'Tues', 'Wed', 'Thu', 'Fri', 'Sat'];
 
+  const classes = [
+    enableWeekendDays && 'enable-weekend-days',
+    leftAlignHeader && 'left-align-header'
+  ]
+    .filter(Boolean)
```

Comment:
Is this doing the same as the `cx from 'classnames` package that we use throughout gather-ui?

```
const classes = cx(`calendar`, {
    'enable-weekend-days': enableWeekendDays,
    'left-align-header': leftAlignHeader,
  });
  
  <div className={classes}>
```

Reply:
Great point - forgot we had access to classnames so I'll get it ammended!